File: PRAK604-2510817210027-Ahmad_Tibrizi.py
kode = input()
pesan = input()
if len(kode) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    temp = ""
    for i, x in zip(kode, pesan):
        if i == x and i != " ":
            temp+="*"
        elif i == x and i == " ":
            temp+=" "
        else:
            temp+="#"
    print(temp)
    bintang = temp.count("*")
    pagar = temp.count("#")
    print(f"* = {bintang}")
    print(f"# = {pagar}")
    if bintang > pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: c/PRAK604-2510817210027-Ahmad_Tibrizi.c
#include <stdio.h>
#include <string.h>

int main() {
    char kode[1000];
    char pesan[1000];

    fgets(kode, 1000, stdin);
    kode[strcspn(kode, "\n")] = 0;

    fgets(pesan, 1000, stdin);
    pesan[strcspn(pesan, "\n")] = 0;

    if (strlen(kode) != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu");
    } else {
        int bintang = 0;
        int pagar = 0;
        int panjang = strlen(kode);

        for (int i = 0; i < panjang; i++) {
            if (kode[i] == pesan[i]) {
                if (kode[i] == ' ') {
                    printf(" ");
                } else {
                    printf("*");
                    bintang++;
                }
            } else {
                printf("#");
                pagar++;
            }
        }
        printf("\n");
        printf("* = %d\n", bintang);
        printf("# = %d\n", pagar);

        if (bintang > pagar) {
            printf("Pesan Asli");
        } else {
            printf("Pesan Palsu");
        }
    }
    printf("\n");
    return 0;
}